perl/p5-hash-moreutils/Portfile

```diff
@@ -4,18 +4,18 @@ PortSystem          1.0
 PortGroup           perl5 1.0
 
 perl5.branches      5.26
-perl5.setup         Hash-MoreUtils 0.05
+perl5.setup         Hash-MoreUtils 0.06
 license             {Artistic-1 GPL}
 maintainers         nomaintainer
 description         Hash::MoreUtils - Provide the stuff missing in Hash::Util
 long_description    ${description}
 
 platforms           darwin
 
-checksums           rmd160  8d30e017153bfcfe38405412e756596c02725da7 \
-                    sha256  5e9c8458457eb18315a5669e3bef68488cd5ed8c2220011ac7429ff983288ab1
+checksums           rmd160  6c24a4fbdab3b05033f80a0d2e7369ab2e2fbfe5 \
+                    sha256  db9a8fb867d50753c380889a5e54075651b5e08c9b3b721cb7220c0883547de8 \
+                    size    9208
 
 if {${perl5.major} != ""} {
-    perl5.use_module_build
     supported_archs noarch
```

Comment:
If this port doesn't contain any binary code and just installs plain text files, it makes sense to put this `supported_archs noarch` globally and not make it depend on whether or not this is a subport.

The port `p5-hash-moreutils` is not actually installing anything (except for maybe a `README` or something) and declaring dependency on `p5.26-hash-moreutils`, so there's no reason why `p5-hash-moreutils` would be `x86_64` and `p5.26-hash-moreutils` would be `noarch`. If anything, it could have been vice versa.

Well, `p5-hash-moreutils` is already declared `noarch` by the PortGroup, so technically declaring the port `noarch` inside or outside this `if` yields the same result, it's just strange to see it inside (somehow conveying the meaning that the main port is not `noarch`).

From what I quickly check the port merely installs one perl module (not binary) and man page. So it would be wrong to remove `supported_arch noarch` completely as was done in the first attempt. But if you are touching this code anyway, you could also put it outside of `if`.

This is such a trivial thing that it would usually not deserve such a long comment. But I wanted to explain because both you and @pmetzger kept asking what this line was good for.

Once again: this PR is good to be merged as is immediately. But if you want, you may remove the `if`.

Reply:
Gotcha. @chrstphrchvz we have our answer.

Reply:
Thanks for the explanation @mojca. Resetting to 99b039f8d8f1ea096c8cd3b0cc860f274927c2a9 (earlier commit).